sdist is missing conftest.py

The resolvelib 1.0.1 sdist package at PyPI is missing the `tests/conftest.py` file.  Without the `tests/conftest.py` file the `test_pin_conflict_with_self` test fails with this error:
```
________________ ERROR at setup of test_pin_conflict_with_self _________________
file $(BUILD_DIR)/tests/test_resolvers.py, line 193
  def test_pin_conflict_with_self(monkeypatch, reporter):
E       fixture 'reporter' not found
>       available fixtures: benchmark, benchmark_weave, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, class_mocker, cov, datadir, doctest_namespace, event_loop, freezer, fs, fs_module, fs_session, mocker, module_mocker, monkeypatch, needs_internet, no_cover, original_datadir, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, script_cwd, script_launch_mode, script_runner, session_mocker, shared_datadir, socket_disabled, socket_enabled, spy_agency, subtests, template_context, testrun_uid, time_machine, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, travis, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, worker_id
>       use 'pytest --fixtures [testpath]' for help on them.

$(BUILD_DIR)/tests/test_resolvers.py:193
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,2 +1,3 @@
 include CHANGELOG.rst LICENSE
 recursive-include src *.pyi py.typed
+recursive-include tests

